data/json/tool_qualities.json

```diff
@@ -207,6 +207,11 @@
     "id": "CHISEL",
     "name": { "str": "chiseling" }
   },
+  {
+    "type": "tool_quality",
+    "id": "BELLOW",
+    "name": { "str": "Bellowing" }
```

Comment:
All other tool qualities have names in lower case.

Reply:
thanks for the very early catch, heh